fastapi_views/exceptions.py

```diff
@@ -31,7 +40,9 @@ def get_status(cls) -> int:
 
     def as_model(self, **kwargs: Any) -> ErrorDetails:
         kwargs = {**self.kwargs, **kwargs}
-        return self.model(detail=self.detail, **kwargs)
+        status = kwargs.get("status", self.get_status())
+        model = self._registry.get(status, self.model)
+        return model(**kwargs)
```

Comment:
Tutaj chyba powinno zostać self.model inaczej troche bez sensu że możesz to nadpisać

Reply:
W przypadku gdy poleci `HTTPException` i ktoś da swój kod, albo będzie to coś z frameorka (np 405) i jest zdefinioany model który ma swój title/type/docs, to chyba lepiej żeby to była jego instancja niż ta bazowa imo?
Pozwoli to dostawać lepsze schematy na podstawie samego `status`